contracts/staking/contracts/test/TestStorageLayout.sol

```diff
@@ -25,6 +25,9 @@ import "../src/interfaces/IStructs.sol";
 
 
 contract TestStorageLayout is
+    MixinDeploymentConstants,
+    Ownable,
+    MixinConstants,
```

Comment:
Yeah, to remove any ambiguities during linearization it will print out the full ancestral tree. Ugly, but it works. I'll create a task to remove frivolous imports for readability to see if we can make this a bit prettier. In the meantime I'll cleanup the ones in this PR.

Reply:
FWIW this is an example demonstrating why explicitly inheriting the ancestral tree works:

```
Ex:
contract A {}
contract B {}
contract C {}
contract D is A, B {}

Suppose now that there is a contract E that will inherit from B{} and D{}
The following follows "most base-like" to "most-derived":
contract E is B, D {}
However, it unfolds to: B{}, A{}, B{}, D{}, E{} - which is wrong because B{} appears both before and after A{}.
The correct contract inheritance for E{} should be:
contract E is A{}, B{}, D{}

So, even though contract E{} does not directly depend on A{} - it still must be included in the dependency list.
```